### PR TITLE
modularized code and app fully functioning

### PR DESCRIPTION
**Pull request by bridgetvon**
Finally got the stack modularized to help with drawer nav, There is a lot more work to do but at least this makes our app entry much cleaner, working on drawer nav in a new branch

### CODE REVIEW

File: screens/MatchScreen.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```
Comment (CountDer3k) [inline screens/MatchScreen.js:1]: If this file is a component, (or used inside of another screen):
move the file into the screenComponents folder.

File: package.json [hunk 2]
```diff
@@ -9,14 +9,19 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "@react-navigation/drawer": "^6.5.0",
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.9.1",
-    "expo": "~46.0.16",
```
Comment (CountDer3k) [inline package.json:14]: Add this line back in.

File: routes/myStack.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import { AppText } from "../assets/AppText";
+import Ionicons from "@expo/vector-icons/Ionicons";
+
+
+const Stack = createNativeStackNavigator();
+
+const FullStack = () => {
+  return (
+    <Stack.Navigator>
+    {/* Move this line below so that Login is the first screen before this is merged into main. */}
```
Comment (CountDer3k) [inline routes/myStack.js:17]: Remove this comment, it's no longer needed

File: App.js [hunk 4]
```diff
@@ -44,55 +39,12 @@ export default function App() {
...
-        />
-        <Stack.Screen
-          name="HorseInfo"
-          component={HorseInformationScreen}
-          options={{
-            headerTitle: (props) => <Text></Text>,
-          }}
-        />
-      </Stack.Navigator>
+      <FullStack />
```
Comment (CountDer3k) [inline App.js:45]: Rename this component to be more descriptive. Possible suggestions:

- ApplicationNavigationStack
- AppNavStack
- RouteNavigationStack

File: screens/MatchScreen.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import { CheckBox } from 'react-native-paper';
+
+const MatchScreen = () => {
+    const [checked, setChecked ] = React.useState(false);
```
Comment (CountDer3k) [inline screens/MatchScreen.js:5]: Conform to Styling Guide #4.

**Review by CountDer3k — CHANGES_REQUESTED** [review]
Small things to change.

**Comment by CountDer3k** [comment]
Update Instructions.txt with all addons you had to install to the project to get the nav bar to work. I saw the json file had new packages in there, so add the commands and links to instructions for how to do that in that file please.

File: App.js (hunk 4 above)
Comment (bridgetvon) [reply App.js:45]: this is a temporary component

File: screens/MatchScreen.js (hunk 5 above)
Comment (bridgetvon) [reply screens/MatchScreen.js:5]: this is a test file for react native paper that I was planning to start but gave up on after once again getting stuck on nav

File: routes/myStack.js (hunk 3 above)
Comment (bridgetvon) [reply routes/myStack.js:17]: will do

File: package.json (hunk 2 above)
Comment (bridgetvon) [reply package.json:14]: got it

File: screens/MatchScreen.js (hunk 1 above)
Comment (bridgetvon) [reply screens/MatchScreen.js:1]: its just a test, should I delete it? Why are you so short with me? do you hate me????

File: App.js (hunk 4 above)
Comment (bridgetvon) [reply App.js:45]: its seperated in another branch, I might make changes to how I'm doing it to get it to work but should be totally different by tomorrow. I'll change it for now tho

File: screens/MatchScreen.js (hunk 1 above)
Comment (CountDer3k) [reply screens/MatchScreen.js:1]: Haha you're funny... If it's a tests I would delete it.

File: screens/MatchScreen.js (hunk 5 above)
Comment (CountDer3k) [reply screens/MatchScreen.js:5]: Ok then we can just remove the paper plugin if you aren't planning on using it.

**Pull request closed without merging**